app/src/test/java/org/oppia/android/app/activity/ActivityIntentFactoriesTest.kt

```diff
@@ -113,11 +116,15 @@ class ActivityIntentFactoriesTest {
       getTopicActivityIntentFactory().createIntent(
         ProfileId.getDefaultInstance(), topicId = "test_topic_id"
       )
+    val args = intent?.getProtoExtra(
+      TopicActivity.TOPIC_ACTIVITY_PARAMS_KEY,
+      TopicActivityParams.getDefaultInstance()
+    )
 
     assertThat(intent).hasComponentClass(TopicActivity::class.java)
-    assertThat(intent).extras().integer(TOPIC_PROFILE_ID_KEY).isEqualTo(0)
-    assertThat(intent).extras().string(TOPIC_ID_KEY).isEqualTo("test_topic_id")
-    assertThat(intent).extras().doesNotContainKey(STORY_ID_KEY)
+    assert(intent.extractCurrentUserProfileId().internalId == 0)
+    assert(args.topicId.equals("test_topic_id"))
+    assert(args.storyId.isEmpty())
```

Comment:
This needs to be assertThat()--ditto below. Please also check everywhere else in the PR that assertThat() is being used instead of assert().

Reply:
Okay.